apps/studio/pages/integrations/vercel/[slug]/deploy-button/new-project.tsx

```diff
@@ -195,8 +193,6 @@ const CreateProject = () => {
             },
             orgSlug: selectedOrganization?.slug,
           })
-
-          await syncEnvs({ connectionId })
```

Comment:
Assuming this has been removed because it now happens automatically / is no longer needed?

Reply:
It was always syncing on creation time, not sure why we ever synced twice 😅